examples/CosmosDB/CosmosDB.Tests.ps1

```diff
@@ -2,12 +2,162 @@ BeforeAll {
   Import-Module Az.InfrastructureTesting
 }
 
+$gremlinAccountName = "gremlin-account-name"
+$gremlinDatabaseName = "gremlin-db-name"
+$mongoAccountName = "mongodb-account-name"
+$mongoDatabaseName = "mongodb-db-name"
+$sqlAccountName = "sql-account-name"
+$sqlDatabaseName = "sql-db-name"
+$rgName = "testrg"
+
+Describe 'Verify Cosmos DB Account with Confirm-AzBPResource' {
+  it 'Should contain a Cosmos DB Account named gremlin-account-name' {
+    #arrange
+    $params = @{
+      ResourceType      = "CosmosDBAccount"
```

Comment:
Same with these resource types but didnt change since they're never used more than once

Reply:
Can you update these to use the `ResourceType` enum, since it was added recently?

This applies to wherever a `ResourceType` is specified in the tests.

Reply:
Disregard, Pester *REALLY* doesn't like enums.